landscape.yml

```diff
@@ -2280,12 +2280,12 @@ landscape:
             twitter: 'https://twitter.com/coreos'
             crunchbase: 'https://www.crunchbase.com/organization/red-hat'
           - item:
-            name: Telekube
+            name: Gravity
             description: >-
-              Telekube combines a production hardened deployment of Kubernetes with Teleport, our multi-region SSH server, so you can effectively manage
-              multiple deployments of Kubernetes applications across various regions, data centers and cloud providers.
+              Bring Kubernetes Down to Earth. Gravity is an upstream Kubernetes packaging solution that takes the drama out of on-premise deployments.
             homepage_url: 'https://gravitational.com/gravity/'
-            logo: ./hosted_logos/gravitational-telekube.svg
+            repo_url: 'https://github.com/gravitational/gravity'
+            logo: ./hosted_logos/logo-gravity.svg
```

Comment:
This build failed because you didn't upload your logo. Please create a version of https://github.com/gravitational/gravity/blob/master/docs/theme/images/logo-gravity.svg that includes the word `Gravity`.

Reply:
updated with link to an logo in the repo with a name.